Directory.Build.targets

```diff
@@ -1,4 +1,23 @@
 <Project>
+  <PropertyGroup Condition=" '$(UseWindowsForms)' == 'true' Or '$(UseWPF)' == 'true' Or '$(ImportWindowsDesktopTargets)' == 'true'  ">
```

Comment:
Shouldn't this go below the section below where you've disabled implicit imports?

Reply:
This doesn't really matters, as long as it comes before FrameworkReference.

Reply:
Anyway, this is not needed. Removing.

In .NET 5.0 the Microsoft.WindowsDesktop.App must be always referenced since this is what '-windows' TFM does.